### ISSUE TITLE
Clarity on the binary files

### ISSUE THREAD

**Issue by BenAkroyd** [jayfk/statuspage#49]
Could you clarify whats in the binary files? Is it the compiled statuspage.py file?

**Comment by jakirkham** [comment]
They are built using [PyInstaller]( http://www.pyinstaller.org/ ). At least, I seem to have been able to do the same locally.

**Comment by jayfk** [comment]
> Could you clarify whats in the binary files? Is it the compiled statuspage.py file?

Yes. They also include the initial template files used to populate the HTML.
